types/abeeway/microTracker/uplink.js

```diff
@@ -0,0 +1,981 @@
+function stepSize(lo, hi, nbits, nresv) {
```

Comment:
add a comment explaining that when changing this, the other types also need to be changed